frontend/public/components/dashboard/project-dashboard/inventory-card.tsx

```diff
@@ -43,33 +46,67 @@ const ProjectInventoryItem = withDashboardResources(
     model,
     mapper,
     useAbbr,
+    additionalResources = [],
   }: ProjectInventoryItemProps) => {
     React.useEffect(() => {
       if (projectName) {
         const resource = createFirehoseResource(model, projectName);
         watchK8sResource(resource);
-        return () => stopWatchK8sResource(resource);
+        if (additionalResources.length) {
```

Comment:
is this `if` needed? It exists as long as `additionalResources` is an array, even empty

Reply:
not needed really, removed